### ISSUE TITLE
nix-daemon fails to start (new user)

### ISSUE THREAD

**Issue by mgidding** [trofi/nix-guix-gentoo#16]
Hello,

I just installed sys-apps/nix, however upon starting the daemon via `systemctl start nix-daemon.service` it fails, with the following error:
```
systemctl status nix-daemon.service 
○ nix-daemon.service - Nix Daemon
     Loaded: loaded (/usr/lib/systemd/system/nix-daemon.service; enabled; vendor preset: disabled)
     Active: inactive (dead)
  Condition: start condition failed at Wed 2022-03-30 11:27:27 CEST; 11s ago
             └─ ConditionPathIsReadWrite=/nix/var/nix/daemon-socket was not met

```

How would I go about solving this? Thank you!

**Comment by trofi** [comment]
Good catch! Thank you for the report!

Reproduced the same failure on fresh install. https://github.com/trofi/nix-guix-gentoo/commit/b39487a59b51c33677c21b9719779ed83339a0da should fix it (backfills https://github.com/NixOS/nix/commit/1bec333788df9b1c9d9c68a428cfc79568d87cf2 change).

Please give it a try. Should be fixed in `2.7.0-r2` and `9999` versions.

**Comment by mgidding** [comment]
I can confirm it runs now, thank you.
